### DISCUSSION TITLE
open-mai is missing orted

### DISCUSSION THREAD

**Discussion by marcpaterno** [Homebrew/discussions#4983] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.25-57-gdc3daf7
ORIGIN: https://github.com/Homebrew/brew
HEAD: dc3daf77a527b5c9da85d90c62f14f57ff6a339e
Last commit: 9 hours ago
Core tap JSON: 15 Dec 17:22 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.HdWzD7qVjV/org.xquartz:0
HOMEBREW_EDITOR: vim
HOMEBREW_MAKE_JOBS: 12
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: dodeca-core 64-bit arm_blizzard_avalanche
Clang: 15.0.0 build 1500
Git: 2.43.0 => /opt/homebrew/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 13.6.2-arm64
CLT: 15.1.0.0.1.1700200546
Xcode: 15.0.1
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

The installed `open-mpi` does not include the binary `orted`. This is required for an R package (`Rmpi1) that uses OpenMPI and for which, on macOS, it is suggested to get OpenMPI through Homebrew.

I do not know enough about the build of OpenMPI to fix this myself. How can I best send a question to the appropriate maintainer(s) to ask if this is feasible?

**Comment by vishalbelsare** [comment]
Did you figure this out? Running into the same issue but on Linux and with Rmpi needing orted.

**Reply by marcpaterno** [reply]
No, I have not found any solution yet.

**Comment by pnavaro** [comment, marked as answer]
`orted` is replaced by `prted` in version 5 of open-mpi. You can try 
```R
install.packages("Rmpi", configure.args = c("ORTED=prted"))
```

**Comment by marcpaterno** [comment]
I have just tried the installation as proposed by pnavaro on my macOS laptop, and it worked fine. Note that one needs, of course, to have OpenMPI already installed; as is suggested in the documentation of Rmpi, I have it installed through Homebrew.

Thanks for the solution!
